src/ID3v2FrameReader.js

```diff
@@ -33,26 +369,22 @@ frameReaderFunctions['APIC'] = function readPictureFrame(
   length: number,
   data: MediaFileReader,
   flags: ?Object,
-  majorVersion?: string
+  id3header?: TagHeader
 ): any {
-  majorVersion = majorVersion || '3';
-
   var start = offset;
   var charset = getTextEncoding(data.getByteAt(offset));
-  switch (majorVersion) {
-    case '2':
+  switch (id3header && id3header.major) {
+    case 2:
     var format = data.getStringAt(offset+1, 3);
     offset += 4;
     break;
 
-    case '3':
-    case '4':
+    case 3:
+    case 4:
+    default:
     var format = data.getStringWithCharsetAt(offset+1, length - 1);
     offset += 1 + format.bytesReadCount;
     break;
-
-    default:
-    throw new Error("Couldn't read ID3v2 major version.");
```

Comment:
Why was this safe guard removed? If the major version isn't one of the supported cases we won't be able to parse it (or it will result in garbage).